config/session.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+class session
+{
+
+    /**
+     * Function to start session
+     *
+     * @access public
+     * @param  void
+     * @return void
+     */
+    function start()
+    {
+        session_start();
+    }
+
+    /**
+     * Function to initialize session fields
+     *
+     * @access public
+     * @param  string $key
+     * @param  string $value
+     * @return void
+     */
+    function init($key, $value)
+    {
+        $_SESSION[$key] = $value;
+    }
+
+    /**
+     * Function to check if session exists
+     *
+     * @access public
+     * @param  void
+     * @return boolean
+     */
+    function checkSession()
+    {
+        if (isset($_SESSION['id']) && !empty($_SESSION['id']))
+        {
+            return TRUE;
+        }
+        else
```

Comment:
else case not required. Can write as
if (isset($_SESSION['id']) && !empty($_SESSION['id']))
{
   return TRUE;
}

return FALSE;